reladomogen/src/main/templates/CommonObjectFactory.jspi

```diff
@@ -49,7 +51,7 @@ under the License.
     public MithraDataObject deserializeFullData(ObjectInput in) throws IOException, ClassNotFoundException
     {
         <% if (wrapper.isTablePerClassSubClass() || wrapper.isTablePerClassSuperClass()) { %>
-            Class dataClass = com.gs.fw.common.mithra.remote.MithraSerialUtil.getDataClassToInstantiate((String) in.readObject());
+            java.lang.Class dataClass = com.gs.fw.common.mithra.remote.MithraSerialUtil.getDataClassToInstantiate((String) in.readObject());
```

Comment:
Fully qualifying classes inside java.lang is usually redundant, but this should avoid name ambiguity when a modeled class is named `Class`.